doc/calling-javascript.md

```diff
@@ -176,6 +176,32 @@ or as
 js.Dynamic.literal("foo" -> 42, "bar" -> "foobar")
 {% endhighlight %}
 
+## Literal object construction using an scala object interface
```

Comment:
Also, consider making this section more about JS facades. Maybe even make the title: "Option objects for native JavaScript libraries"